common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_action.yaml

```diff
@@ -100,7 +100,14 @@ eActionOp_CONTROL:
   ACTION_CONTROL_STEERING_EVENT_PROBE_REQ_NOTIFICATION : 133
   ACTION_CONTROL_STEERING_EVENT_AUTH_FAIL_NOTIFICATION : 134
 
-  ACTION_CONTROL_ENUM_END: 135
+  ACTION_CONTROL_CHANNEL_SCAN_TRIGGER_SCAN_REQUEST : 140
+  ACTION_CONTROL_CHANNEL_SCAN_TRIGGER_SCAN_RESPONSE : 141
+  ACTION_CONTROL_CHANNEL_SCAN_TRIGGERED_NOTIFICATION : 142
+  ACTION_CONTROL_CHANNEL_SCAN_RESULTS_NOTIFICATION : 143
+  ACTION_CONTROL_CHANNEL_SCAN_ABORT_NOTIFICATION : 144
+  ACTION_CONTROL_CHANNEL_SCAN_FINISHED_NOTIFICATION : 145
+
+  ACTION_CONTROL_ENUM_END: 150
```

Comment:
Why not 146?

Reply:
changed

Reply:
I think it make sense to leave "space" for future actions. this will make it easier to add new ones without rewriting the end enum.

Reply:
> I think it make sense to leave "space" for future actions. this will make it easier to add new ones without rewriting the end enum.

In fact, the _END values are not used anywhere. Why do we add them?